ci-operator/jobs/RedHatQE/firewatch/RedHatQE-firewatch-main-postsubmits.yaml

```diff
@@ -1,10 +1,10 @@
 postsubmits:
-  CSPI-QE/firewatch:
+  RedHatQE/firewatch:
   - agent: kubernetes
     always_run: true
     branches:
     - ^main$
-    cluster: build05
+    cluster: build03
```

Comment:
Which cluster is this

Reply:
The build cluster that is used. This is automatically decided by `make update`.